### ISSUE TITLE
Docs? Examples?

### ISSUE THREAD

**Issue by doxxon** [illuminate/socialite#6]
Are there any docs on how to use this package with L4?

**Comment by GrahamCampbell** [comment]
No. This package is depreciated. @taylorotwell Can you close this please?

**Comment by mastacheata** [comment]
It has never been supported at al and you should probably see it more as a demonstration than a readymade library.
But that doesn't mean it no longer works. It's running the same way as with L4 even in L4.1

**Comment by EDDYMENS** [comment]
why is this package deprecated ? curious... :thinking_face:

**Comment by mastacheata** [comment]
Because it's been superseeded by this one: https://github.com/laravel/socialite

Apparently this was the first attempt for L4, but due to time constraints and interest of developers it never matured to a point where it was easy to use and stable and what not. 😉

**Comment by EDDYMENS** [comment]
AAH ok

**Comment by EDDYMENS** [comment]
And oh yh why is that most illuminate packages do not docs ? or am not looking hard enough ..

**Comment by mastacheata** [comment]
Usually because they are meant for internal use by Laravel only.
Also you will find that good developers will barely write any documentation. They usually know about the importance of documentation, but will neglect it anyways.

**Comment by EDDYMENS** [comment]
oohk so its not that it ain't needed, just that others who contribute can grasp what is happening from source .... cool.

**Comment by browner12** [comment]
ping @taylorotwell @GrahamCampbell @themsaid any chance we could get rid of this repo, or at least mark it as deprecated in the description?

**Comment by GrahamCampbell** [comment]
I think it's obvious enough it's deprecated since there's been no activity in 5 years?

**Comment by browner12** [comment]
Does it make sense to remove it?

Can we mark as abandoned on Packagist?  I know these are minor things, but they help eliminate confusion.

**Comment by EDDYMENS** [comment]
I agree with marking it as deprecated at least.
